src/mono/wasm/debugger/BrowserDebugProxy/MemberReferenceResolver.cs

```diff
@@ -242,33 +257,61 @@ async Task<JObject> ResolveAsLocalOrThisMember(string name)
                     return null;
                 }
 
-                JToken objRet = valueOrError.Value.FirstOrDefault(objPropAttr => objPropAttr["name"].Value<string>() == nameTrimmed);
+                JToken objRet = valueOrError.Value.FirstOrDefault(objPropAttr => objPropAttr["name"].Value<string>() == name);
                 if (objRet != null)
                     return await GetValueFromObject(objRet, token);
 
                 return null;
             }
 
-            async Task<JObject> ResolveAsInstanceMember(string expr, JObject baseObject)
+            async Task<JObject> ResolveAsInstanceMember(ArraySegment<string> parts, JObject baseObject, bool hasNullCondition)
             {
+                // if the prevPart has "?" at the end: true
+                bool hasResolvedObjNullCondition = hasNullCondition;
+                bool isPrevPartNull = false;
```

Comment:
This is not needed, AFAIU.

Reply:
You're right. I replaced it with earlier update of `throwOnNullReference` and continuing.